test/unit/generator/content/get-address.test.js

```diff
@@ -170,14 +170,14 @@ describe('get address', () => {
       postcode: 'NE1 1AA'
     }
     const result = getAddress(businessName, address)
-    expect(result.absolutePosition.x).toBe(28.35)
+    expect(result.absolutePosition.x).toBe(0)
   })
 
   test('returns correct location y axis', () => {
     const address = {
       postcode: 'NE1 1AA'
     }
     const result = getAddress(businessName, address)
-    expect(result.absolutePosition.y).toBe(99.225)
+    expect(result.absolutePosition.y).toBe(70.875)
```

Comment:
why isn't this the defined 25 in getAddress?

Reply:
Style measurement unit is millimeters while pdf measurement unit is points. Conversion from millimeters to Points is in multiple of 2.835.

https://github.com/DEFRA/ffc-pay-statement-generator/blob/main/app/generator/conversion.js